test/fixtures/thermostats.yml

```diff
@@ -4,8 +4,9 @@
 # model remove the '{}' from the fixture names and add the columns immediately
 # below each fixture, per the syntax in the comments below
 #
-one: {}
+thermostat_one: {}
 # column: value
 #
-two: {}
+thermostat_two:
```

Comment:
Are you sure this rims_id works as expected? I'm getting an error in the controller test because the view is trying to get the pins of a nil object.

If it is legit, then the problem may be the Thermostat's own subsystems.